sonarts-core/tests/rules/noInMisuseRule/noInMisuseRule.lint.ts

```diff
@@ -0,0 +1,57 @@
+function nokFunc1() {
+  let arr = ["a", "b", "c"];
+
+  let expectedValue = "b";
+  if (expectedValue in arr) {
+//    ^^^^^^^^^^^^^^^^^^^^    {{Use "indexOf" or "includes" (available from ES2016) instead.}}
+    return expectedValue + " found in the array";
+  } else {
+    return expectedValue + " not found";
+  }
+}
+
+function nokFunc2() {
```

Comment:
@andrea-guarino-sonarsource Don't perceive RSPEC desc as unit test, it's just starting point. (drop this function and move only `"1" in arr` to previous func)